xbmc/GUIInfoManager.cpp

```diff
@@ -6872,6 +6872,20 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     <p><hr>
 ///     @skinning_v21 **[New Infolabel]** \link ListItem_SongVideoURL `ListItem.SongVideoURL`\endlink
 ///   }
+///   \table_row3{   <b>`ListItem.VideoWidth`</b>,
+///                  \anchor ListItem_VideoWidth
+///                  _string_,
+///     @return String containing width of video in pixels, empty if unknown.
```

Comment:
This will break the doxygen table, please escape commas 
```suggestion
///     @return String containing width of video in pixels\, empty if unknown.
```

Reply:
A dash OK?

Reply:
no, the commas are used to specify columns on the table: https://xbmc.github.io/docs.kodi.tv/master/kodi-base/d5/d11/modules__infolabels_boolean_conditions.html. So, `table_row3` has 3 columns, each time you add a comma without being escaped it will simply assume you're creating a new column. The `\` escapes the comma, it's not rendered on the page at all

Reply:
well I see it now, if you drop the comma that works as well :)